packages/graphback-runtime/src/resolvers/LayeredRuntimeResolverGen.ts

```diff
@@ -36,44 +35,45 @@ export class LayeredRuntimeResolverGenerator {
       if (resolverElement.config.disableGen) {
         continue;
       }
-
+      const objectName = resolverElement.name.toLowerCase();
       if (resolverElement.config.create) {
         const resolverCreateField = getFieldName(resolverElement.name, GraphbackOperationType.CREATE);
         // tslint:disable-next-line: no-any
```

Comment:
We disable this rule in so many places. Do you think we should disable the rule as this line is littered everywhere?

Reply:
I think we had talked about this in runtime PR. Generally ANY shoud not be used at all. 
This is very very specific exception where we declare runtime layer (and our code) will be actually an generic execution code so we do not benefit from typings at all (or even can do typings here)